cohort_banking_initiator_js/src/main.ts

```diff
@@ -71,6 +111,16 @@ new Promise(async (resolve) => {
         queue,
         fnFinish,
     )
-    await app.init()
+    try {
```

Comment:
this is example where caller can deal with specific errors. I skipped else statement here.